components/CongresspersonInfoTab/index.js

```diff
@@ -0,0 +1,153 @@
+import * as CUI from '@chakra-ui/react';
+
+const mapEducationLevelLabel = {
+  BASICA: 'Primaria Incompleta',
+  BASICA_PRIMARIA: 'Primaria Completa',
+  BASICA_SECUNDARIA: 'Secundaria Completa',
+  NO_UNIVERSITARIA: 'Técnico superior',
+  UNIVERSITARIA: 'Universitaria',
+  POSTGRADO: 'Postgrado',
+};
+
+export default function CongresspersonInfoTab({ congresspersonData }) {
+  const congresspersonTabs = [
+    {
+      id: 0,
+      label: 'Información personal',
+      content: (
+        <TabPanelLayout title="Información personal">
+          <LabelWithText label="EDAD" text={`${congresspersonData.age} años`} />
+          <LabelWithText
+            label="LUGAR DE NACIMIENTO"
+            text={congresspersonData.birthPlace}
+          />
+          <LabelWithText
+            label=" LUGAR DE RESIDENCIA"
```

Comment:
Es necesario ese espacio?
```suggestion
            label="LUGAR DE RESIDENCIA"
```